### PR TITLE
tikv-ctl: add short name for raftdb

### PR DESCRIPTION
**Pull request by javaforfun**
currently:  
```shell
$./tikv-ctl -d data/db/ --raftdb data/raft/  raft region
error: Found argument '--raftdb' which wasn't expected, or isn't valid in this context

$ ./tikv-ctl -d data/db/ <raftdb> data/raft/  raft region
-bash: raftdb: No such file or directory

$ ./tikv-ctl -d data/db/  data/raft/  raft region
```
we should make -r work
```shell
$./tikv-ctl -d data/db/ -r data/raft/  raft region
```

### CODE REVIEW

**Comment by BusyJay** [comment]
Can it take a default value?

File: src/bin/tikv-ctl.rs [hunk 1]
```diff
@@ -249,8 +250,12 @@ fn main() {
     };
     let db_path = matches.value_of("db").unwrap();
     let db = util::rocksdb::open(db_path, ALL_CFS).unwrap();
-    let raft_db_path = matches.value_of("raftdb").unwrap();
-    let raft_db = util::rocksdb::open(raft_db_path, &[]).unwrap();
+    let raft_db = if let Some(raftdb_path) = matches.value_of("raftdb") {
+        util::rocksdb::open(raftdb_path, &[CF_DEFAULT]).unwrap()
+    } else {
+        let raftdb_path = db_path.to_owned() + "../raft";
```
Comment (siddontang) [inline src/bin/tikv-ctl.rs]: do you test it?

**Comment by siddontang** [comment]
Rest LGTM

File: src/bin/tikv-ctl.rs (hunk 1 above)
Comment (javaforfun) [reply src/bin/tikv-ctl.rs]: of course i've tested it

**Pull request merged**